lang/po/Firmware_nl.po

```diff
@@ -425,230 +451,363 @@ msgstr ""
-msgstr "FOUT: Filamentsensor reageert niet, controleer de verbinding."
-
 #. MSG_EJECT_FILAMENT c=17
-#: ../../Firmware/messages.cpp:56 ../../Firmware/ultralcd.cpp:5164
-#: ../../Firmware/ultralcd.cpp:5573
+#: ../../Firmware/messages.cpp:60 ../../Firmware/ultralcd.cpp:5137
+#: ../../Firmware/ultralcd.cpp:5575
 msgid "Eject filament"
 msgstr "Fil. uitwerpen"
 
-#. MSG_EJECTING_FILAMENT c=20
-#: ../../Firmware/mmu.cpp:1076
+#. MSG_PROGRESS_EJECT_FILAMENT c=20
+#. @@todo duplicate
+#: ../../Firmware/mmu2_progress_converter.cpp:27
+#: ../../Firmware/mmu2_progress_converter.cpp:58
 msgid "Ejecting filament"
 msgstr "Fil. word ontladen"
 
 #. MSG_SELFTEST_ENDSTOP c=16
-#: ../../Firmware/ultralcd.cpp:6993
+#: ../../Firmware/ultralcd.cpp:6984
 msgid "Endstop"
 msgstr "Eindstop"
 
 #. MSG_SELFTEST_ENDSTOP_NOTHIT c=20
-#: ../../Firmware/ultralcd.cpp:6998
+#: ../../Firmware/ultralcd.cpp:6989
 msgid "Endstop not hit"
 msgstr "Endstop niet geraakt"
 
 #. MSG_SELFTEST_ENDSTOPS c=20
-#: ../../Firmware/ultralcd.cpp:6984
+#: ../../Firmware/ultralcd.cpp:6975
 msgid "Endstops"
 msgstr "Eindstops"
 
+#. MSG_PROGRESS_ENGAGE_IDLER c=20
+#: ../../Firmware/mmu2_progress_converter.cpp:9
+#: ../../Firmware/mmu2_progress_converter.cpp:35
+#: ../../Firmware/mmu2_progress_converter.cpp:45
+msgid "Engaging idler"
+msgstr "Koppeln rolspel"
+
 #. MSG_EXTRUDER c=17
-#: ../../Firmware/Marlin_main.cpp:8605 ../../Firmware/messages.cpp:30
-#: ../../Firmware/ultralcd.cpp:3503
+#: ../../Firmware/Marlin_main.cpp:3519 ../../Firmware/Tcodes.cpp:35
+#: ../../Firmware/Tcodes.cpp:51 ../../Firmware/messages.cpp:30
+#: ../../Firmware/ultralcd.cpp:3512
 msgid "Extruder"
 msgstr "Extruder"
 
 #. MSG_EXTRUDER_FAN_SPEED c=16
-#: ../../Firmware/messages.cpp:35 ../../Firmware/ultralcd.cpp:1134
-#: ../../Firmware/ultralcd.cpp:7329
+#: ../../Firmware/messages.cpp:35 ../../Firmware/ultralcd.cpp:1144
+#: ../../Firmware/ultralcd.cpp:7319
 msgid "Extruder fan:"
 msgstr "Extruder fan:"
 
 #. MSG_INFO_EXTRUDER c=18
-#: ../../Firmware/ultralcd.cpp:1730
+#: ../../Firmware/ultralcd.cpp:1722
 msgid "Extruder info"
 msgstr "Extruder info"
 
 #. MSG_FSENSOR_AUTOLOAD c=13
-#: ../../Firmware/messages.cpp:44 ../../Firmware/ultralcd.cpp:4237
-#: ../../Firmware/ultralcd.cpp:4245 ../../Firmware/ultralcd.cpp:4256
-#: ../../Firmware/ultralcd.cpp:4258
+#: ../../Firmware/messages.cpp:45 ../../Firmware/ultralcd.cpp:4230
+#: ../../Firmware/ultralcd.cpp:4237
 msgid "F. autoload"
 msgstr "F. autoladen"
 
+#. MSG_FSENSOR_JAM_DETECTION c=13
+#: ../../Firmware/messages.cpp:46 ../../Firmware/ultralcd.cpp:4232
+#: ../../Firmware/ultralcd.cpp:4239
+msgid "F. jam detect"
+msgstr ""
+
+#. MSG_FSENSOR_RUNOUT c=13
+#: ../../Firmware/messages.cpp:44 ../../Firmware/ultralcd.cpp:4229
+#: ../../Firmware/ultralcd.cpp:4236
+msgid "F. runout"
+msgstr "FS. uitloop"
+
+#. MSG_TITLE_FIL_ALREADY_LOADED c=20
+#: ../../Firmware/mmu2/errors_list.h:148 ../../Firmware/mmu2/errors_list.h:186
+msgid "FILAMENT ALREADY LOA"
```

Comment:
Is the EN message deliberately truncated?

Reply:
Yes it is